router/dataplane.go

```diff
@@ -2848,9 +2779,7 @@ func (p *scionPacketProcessor) currentHopPointer( /*@ ghost ubScionL []byte @*/
 // @ requires  acc(p.scionLayer.Mem(ubScionL), R3)
 // @ requires  p.scionLayer.ValidPathMetaData(ubScionL)
 // @ requires  p.path == p.scionLayer.GetPath(ubScionL)
-// @ requires  acc(&p.buffer, R50) && p.buffer.Mem()
-// @ requires  acc(&p.buffWithFullPerm)
-// @ requires  p.buffWithFullPerm
+// @ requires  acc(&p.buffer, R50) && p.buffer != nil && p.buffer.Mem()
```

Comment:
I find it surprising that we never need to pass permissions to `Bytes(p.buffer, ..., ...)`